operator/src/main/java/oracle/kubernetes/weblogic/domain/model/InitializeDomainOnPV.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2023, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.model;
+
+import com.google.gson.annotations.SerializedName;
+import oracle.kubernetes.json.Description;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+public class InitializeDomainOnPV {
+
+  @Description("Describes the configuration for creating a PersistentVolume for `Domain in PV``. The operator will"
+      + " not recreate or update the PersistentVolume if it already exists. More info:"
+      + " https://kubernetes.io/docs/concepts/storage/persistent-volumes")
+  PersistentVolume persistentVolume;
+
```

Comment:
This is optional and together with pv is where the shared domain exists.  Also,  if this is specified, it must match one volume under serverPod

Reply:
Changed.